src/factory.js

```diff
@@ -2,7 +2,7 @@
  * @license
  * Blockly Demos: Block Factory
  *
- * Copyright 2016 Google Inc.
+ * Copyright 2017 Google Inc.
```

Comment:
To clarify, I only change the copyright if I have created the file from scratch (ha), correct? And I would leave it be if I only made edits (whether major or minor)?